tests/integration_tests/test_sql_task.py

```diff
@@ -333,6 +333,89 @@ def test_sql_access_request_task(db, policy, integration_postgres_config) -> Non
         > 0
     )
 
+# todo- add mssql test
+
+@pytest.mark.integration
+def test_mysql_access_request_task(db, policy, connection_config_mysql) -> None:
+
+    privacy_request = PrivacyRequest(
+        id=f"test_mysql_access_request_task_{random.randint(0, 1000)}"
+    )
+
+    v = graph_task.run_access_request(
+        privacy_request,
+        policy,
+        integration_db_graph("my_mysql_db_1"),
+        [connection_config_mysql],
+        {"email": "customer-1@example.com"},
+    )
+
```

Comment:
Would you mind adding a separate postman folder with a MySQL collection for local end-to-end testing? Ideally it would create a connection, add the secrets, and create a MySQL dataset. I moved the MSSQL calls into their own folder, so the folder with the "minimum calls to create an access request" doesn't get too bloated.

Reply:
Definitely, good call!